### DISCUSSION TITLE
DDS sync_yanked plugin, nothing happens

### DISCUSSION THREAD

**Discussion by hakan-demirli** [sxyazi/yazi#1265] [1. Q&A] [answered]
I want to enable DDS sync_yanked plugin. I just copy paste the given code to my `init.lua`. However, It looks like it is not detected/working.

How can I debug this? I have tried `yazi --local-events=yank,update_yank > /tmp/hi` and `yazi --remote-events=yank,update_yank > /tmp/hi` but it only prints `yank` events and there is no `update_yank` entry.

When I yank in one and the green box appears I expect the same green box to appear on the other yazi instance too.

[Kooha-2024-07-06-02-31-12.webm](https://github.com/sxyazi/yazi/assets/78746991/012f4c4a-ce85-471e-a43d-1ec7aa6fe1c6)

Do I need to `import` init.lua or give its path to yazi.toml or smth?

My `init.lua`:
```lua
require("session"):setup({
  sync_yanked = true,
})
```
This is my config structure.
```
├── init.lua
├── keymap.toml
├── plugins
│   ├── jump-to-char.yazi
│   │   └── init.lua
│   └── smart-enter.yazi
│       └── init.lua
├── theme.toml
└── yazi.toml
 ```

<details>
<summary>`yazi --debug`</summary>

```
Yazi
    Version: 0.2.5 (VERGEN_IDEMPOTENT_OUTPUT 1980-01-01)
    OS: linux-x86_64 (unix)
    Debug: false

Emulator
    Emulator.via_env: ("xterm-kitty", "")
    Emulator.via_csi: Ok(Kitty)
    Emulator.detect: Kitty

Adaptor
    Adaptor.matches: Kitty

Desktop
    XDG_SESSION_TYPE: Some("wayland")
    WAYLAND_DISPLAY: Some("wayland-1")
    DISPLAY: Some(":1")

SSH
    shared.in_ssh_connection: false

WSL
    /proc/sys/fs/binfmt_misc/WSLInterop: false

Variables
    SHELL: Some("/run/current-system/sw/bin/bash")
    EDITOR: Some("hx")
    ZELLIJ_SESSION_NAME: None
    YAZI_FILE_ONE: None
    YAZI_CONFIG_HOME: None

file(1)
    Version: Ok(Output { status: ExitStatus(unix_wait_status(0)), stdout: "file-5.45\nmagic file from /nix/store/13b54bc6wphsvp93852n66fnl280llzd-file-5.45/share/misc/magic\n", stderr: "" })

Text Opener
    default: Some(Opener { run: "$EDITOR \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })
    block: Some(Opener { run: "$EDITOR \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })

tmux
    TMUX: false

Ueberzug++
    Version: Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })


```
</details>

<details>
<summary>`yazi.log`</summary>

  2024-07-05T19:39:53.559450Z ERROR yazi_scheduler::preload::preload: Preload task `mime` returned 2
    at yazi-scheduler/src/preload/preload.rs:42

</details>

**Comment by sxyazi** [comment, marked as answer]
Try to build from latest source code and `rm ~/.local/state/yazi/.dds`, and make sure to kill and restart all Yazi processes

**Reply by hakan-demirli** [reply]
Yep. `nix run github:sxyazi/yazi` works as expected. Looks like the release version `0.2.5` in `nixpkgs-unstable`does not have dds.
